articles/connections/passwordless/_init-auth0js.md

```diff
@@ -1,13 +1,11 @@
 Construct a new instance of the Auth0 client as follows:
 
 ```html
-<script src="${auth0js_url}"></script>
+<script src="${auth0js_urlv8}"></script>
 <script type="text/javascript">
-  'use strict';
-  const auth0 = new Auth0({
-    clientID: '${account.clientId}',
-    domain: '${account.namespace}'<% if (withCallbackURL) { %>,
-    callbackURL: '${account.callback}'<%}%>
+var webAuth = new auth0.WebAuth({
+	domain: '${account.namespace}',
+    clientID: '${account.clientId}'
   });
   //...
```

Comment:
@angelaLazar This doesn't look complete. For example, this script is part of this doc https://auth0-docs-content-pr-3675.herokuapp.com/docs/connections/passwordless/spa-sms#use-your-own-ui but if you see the next scripts have not been updated. Also, this doc is accompanied by a sample https://github.com/auth0-samples/auth0-jquery-passwordless-sample which still uses auth0.js 6 (https://github.com/auth0-samples/auth0-jquery-passwordless-sample/blob/master/custom-sms.html#L47) We cannot have the doc using one version and the sample using another, it's inconsistent and confusing to customers.

@jeffreylees can you follow up with someone for this? if we want to update the passwordless scripts (which we should since the API now uses v8) the samples have to be updated as well (or removed from the doc). Will someone from engineering do it?